graphql-api.cabal

```diff
@@ -35,11 +35,11 @@ library
     , graphql
     , text
   exposed-modules:
-      GraphQL.Definitions
       GraphQL.Input
       GraphQL.Output
       GraphQL.Schema
-      GraphQL.TypeApi
+      GraphQL.TypedApi
+      GraphQL.TypedSchema
```

Comment:
How's this?

* `GraphQL.API` -- How to define a GraphQL API (currently TypedSchema)
* `GraphQL.Server` -- How to implement handlers for a GraphQL API (currently TypedApi)
* `GraphQL.Literals` -- Literal values in GraphQL (currently Value)
* `GraphQL.Internal`
  * `Input`
  * `Output`
  * `Schema`
  * `Validation`